packages/contracts-bedrock/test/invariants/interop/FuzzTest.t.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Setup, Constants, ConfigType } from "./Setup.sol";
+
+contract FuzzTest is Setup {
+    function test_place_holder() public {
```

Comment:
We need to tests that the code length of the deployed contracts is greater zero and/or that the initialized vars are correct to make sure the setup is working

Reply:
Added the code.length check.